apps/checkout/src/Root.tsx

```diff
@@ -30,7 +30,7 @@ const saleorClient = createSaleorClient({
 });
 
 export const Root = () => {
-  const orderToken = getQueryVariables().orderToken;
+  const orderToken = getQueryVariables().order;
```

Comment:
This was intentional because token will become id at some point, and it will be used not just on the checkout itself but also on storefront if they choose to provide their own order page. I took a look at how these are usually handled in production, and it seems like they always opt for a shorthand (orderId -> order)